doc/Language/operators.pod6

```diff
@@ -1779,7 +1779,7 @@ also modify the other.
 X<|smartmatch operator>
 =head2 infix C«~~»
 
-The X<smart-match operator> aliases the left-hand side to C<$_>, then evaluates
+The X<smartmatch operator> aliases the left-hand side to C<$_>, then evaluates
```

Comment:
You might want to leave this one as it is, since it's used for indexing. If you don't, just eliminate the `X<>` since it's already indexed a few lines above.